resources/js/Pages/Asisten.vue

```diff
@@ -31,7 +31,7 @@
               </div>
               <div class="w-1 h-3/4 my-auto bg-yellow-500"/>
               <div class="flex w-1/2min mr-auto h-full pt-1 items-center font-overpass-bold text-2xl text-white ml-4">
-                <span class="whitespace-pre-wrap">Rp. {{ gajiAsisten }} | Pajak kas: {{ taxRate }} * gaji</span> 
+                <span class="whitespace-pre-wrap">Rp. {{ gajiAsisten }} | Pajak kas: {{ taxRate*100 }}% x gaji</span> 
```

Comment:
This thing has something to do with the backend also, causes the taxRate not showing. We’ll get them next time.

Reply:
okay i'll checkup again this backend, but in local, this is working properly